add_cron.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+(crontab -l; echo "31 6 * * * /app/backup_cron.sh >> /var/log/cron.log 2>&1") | sort -u | crontab -
```

Comment:
could just move this into the captain-definition file?

Reply:
trying to move it to the captain-definition